src/components/BulkEditList/BulkEditListResult/Preview/ErrorsAccordion/ErrorsAccordion.js

```diff
@@ -68,6 +68,7 @@ const ErrorsAccordion = ({
               columnMapping={columnMapping}
               formatter={resultsFormatter}
               visibleColumns={visibleColumns}
+              maxHeight={window.innerHeight * 0.4}
```

Comment:
Please move this coeficient to separate variable, because it's unclear why it's 0.4